test/core/hub/integration/Cases.t.sol

```diff
@@ -153,7 +153,52 @@ contract TestCases is BaseTest {
         _assertEqAccountValue(poolId, ASSET_USDC_ACCOUNT, true, 320 * poolDecimals);
         _assertEqAccountValue(poolId, EQUITY_ACCOUNT, true, 400 * poolDecimals);
         _assertEqAccountValue(poolId, GAIN_ACCOUNT, true, 40 * poolDecimals);
-        _assertEqAccountValue(poolId, LOSS_ACCOUNT, false, 120 * poolDecimals);
+        _assertEqAccountValue(poolId, LOSS_ACCOUNT, true, 120 * poolDecimals);
```

Comment:
Loss is debit-normal and debited on value decrease of an asset, so it is positive